Dockerfile

```diff
@@ -13,7 +12,29 @@ COPY --chown=1001:0 . /tmp/src
 
 USER 1001
 
-ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true QMAKE=/usr/lib64/qt5/bin/qmake DISABLE_ASSET_COMPILATION=true
+ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true QMAKE=/usr/lib64/qt5/bin/qmake DISABLE_ASSET_COMPILATION=true BUNDLE_WITHOUT=development:test:tableau
+
+# Install the dependencies
+RUN /usr/libexec/s2i/assemble
+
+CMD ["/opt/app-root/src/entrypoint.sh"]
+
+FROM compliance-backend as compliance-tableau
+
+USER 0
+
+RUN subscription-manager attach && \
+    subscription-manager repos --enable "codeready-builder-for-rhel-8-$(arch)-rpms" && \
+    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1).noarch.rpm && \
+    yum install -y https://apache.jfrog.io/artifactory/arrow/centos/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm && \
+    yum install -y parquet-glib-devel
+
+# Make an empty file to trick s2i to believe that it's moving the source files
+RUN install -o 1001 -g 0 /tmp/src/placeholder
```

Comment:
Can we run a bundle install or similar to avoid doing these sort of hack please ? s2i after all does that (among other things) 
see https://github.com/sclorg/s2i-ruby-container/blob/master/3.0/s2i/bin/assemble for reference

Reply:
Yes, I can look into that